sites/public/src/components/account/StatusItem.tsx

```diff
@@ -24,7 +24,7 @@ interface StatusItemProps {
 
 const StatusItem = (props: StatusItemProps) => {
   return (
-    <Card.Section className={accountStyles["account-card-applications-section"]}>
+    <Card.Section className={applicationsViewStyles["account-card-applications-section"]}>
```

Comment:
Is it preferred to duplicate this class instead of this crossover @emilyjablonski ?

Reply:
This lgtm!